build.gradle

```diff
@@ -20,6 +22,10 @@ repositories {
     mavenCentral()
 }
 
+gitProperties {
+    gitPropertiesResourceDir = "${project.rootDir}/checker/src/main/resources"
```

Comment:
Is there a reason to put this under `checker` instead of under `framework`?
The use is in `framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java` so putting the related file under `framework` would be more consistent.